cmd/tabledemo/main.go

```diff
@@ -0,0 +1,80 @@
+package main
```

Comment:
This demo as well as the "table" widget should be removed. I think this was pulled in by mistake. I wrote this table before Fyne had a first-party table widget, so it's not needed now methinks.